Sanic performance options documentation (#8201)
getsentry/sentry-docs: src/platforms/python/common/integrations/sanic/index.mdx

app = Sanic(__name__)
```

### Configuration options

> If you are using performance monitoring, you can configure the Sanic integration to not send transactions for requests which have resulted in certain HTTP statuses using the `unsampled_statuses` option.

Reviewer: ```suggestion
If you're using performance monitoring, you can configure the Sanic integration to not send transactions for requests that resulted in certain HTTP statuses using the `unsampled_statuses` option.
```

> By default, the Sanic integration generates transactions for all requests, except requests which result in a 404 HTTP status. If you wish to generate transactions for all HTTP statuses, including 404, or change the HTTP statuses which have no transactions generated, you can manually instantiate the Sanic integration when initializing the Sentry SDK, like so:

Reviewer: ```suggestion
By default, the Sanic integration generates transactions for all requests, except requests that result in a 404 HTTP status. To generate transactions for all HTTP statuses, including 404, or change which HTTP statuses don't generate transactions, you can manually instantiate the Sanic integration when initializing the Sentry SDK, like so:
```

```python
from sanic import Sanic
import sentry_sdk
from sentry_sdk.integrations.sanic import SanicIntegration

sentry_sdk.init(
    dsn="___PUBLIC_DSN___",

    # Configure the Sanic integration so that we
    # generate transactions for all HTTP statuses,
    # including 404
    integrations=[SanicIntegration(unsampled_statuses=None)]

    # Set traces_sample_rate to 1.0 to capture 100%
    # of transactions for performance monitoring.
    traces_sample_rate=1.0,
)
```

> If you wish to specify a different set of HTTP statuses for which no transactions should be generated, set `unsampled_statuses` to a set containing the HTTP statuses which should have no transactions generated. For example, if you wish to have no transactions for requests resulting in a 200 or 404 HTTP status, use `SanicIntegration(unsampled_statuses={200, 400})`.

Reviewer: ```suggestion
To specify a different set of HTTP statuses for which no transactions should be generated, set `unsampled_statuses` to a set containing the HTTP statuses that don't generate transactions. For example, if you wish to have no transactions for requests resulting in a 200 or 404 HTTP status, use `SanicIntegration(unsampled_statuses={200, 400})`.
```
